config/locales/cy.yml

```diff
@@ -260,6 +262,9 @@ cy:
         few:
         many:
         other: Nodiadau siarad
+      service_sign_in:
+        one: Gwasanaeth Mewngofnodi
```

Comment:
I had a conversation with Tom about this translation when he asked me to sense check this as a slug.  I think it's incorrect as it translates as "Sign in service".  I think it should be "Mewngofnodi i wasanaeth" which is literally "Sign in to service" and I think more correct.

Reply:
Thanks! I'll update and fix up. There's also a failing test that needs to fixed.

Reply:
Speaking of the slug. We haven't considered translating "sign-in" in the base_path!

I've agreed with Tom, that's it's not as important as the rest of the translated content, and that we can add it as a stretch task for next week.